package.json

```diff
@@ -7,11 +7,11 @@
     "commit-amend-build": "scripts/commit-amend-build.sh",
     "prebuild": "rimraf dist/**",
     "dev": "cross-env NODE_ENV=development BABEL_ENV=rollup rollup -w -c -o dist/purify.js",
-    "build": "cross-env run-p build:umd build:umd:min build:es build:cjs",
-    "build:umd": "cross-env NODE_ENV=development BABEL_ENV=rollup rollup -c -f umd -o dist/purify.js",
-    "build:umd:min": "cross-env NODE_ENV=production BABEL_ENV=rollup rollup -c -f umd -o dist/purify.min.js",
-    "build:es": "cross-env NODE_ENV=development BABEL_ENV=rollup rollup -c -f es -o dist/purify.es.js",
-    "build:cjs": "cross-env NODE_ENV=development BABEL_ENV=rollup rollup -c -f cjs -o dist/purify.cjs.js",
+    "build": "rollup -c",
+    "build:umd": "rollup -c -f umd -o dist/purify.js",
+    "build:umd:min": "rollup -c -f umd -o dist/purify.min.js -p terser",
+    "build:es": "rollup -c -f es -o dist/purify.es.js",
+    "build:cjs": "rollup -c -f cjs -o dist/purify.cjs.js",
```

Comment:
I kept the individual build commands and only changed the internals.
Minimized env settings that add complexity.